defaults/main.yml

```diff
@@ -41,7 +41,7 @@ owncloud_edition: 'community'
 #
 # ownCloud version in the format ``major_release.minor_release`` or ``testing``.
 # See http://download.opensuse.org/repositories/isv:/ownCloud:/community:/ for the full list.
-owncloud_release: 8.1
+owncloud_release: 9.0
```

Comment:
I'm not sure if that variable is even needed anymore... It's used in 1 place to check if version is 8.0, but it's not autodetected.